notebooks/dataAccess/siphon-RemoteOpen.ipynb

```diff
@@ -0,0 +1,378 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = data_file.read(100)\n",
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can change our position in the file using `seek`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_file.seek(0) # go to start of file\n",
+    "print(data_file.read(4)) # print first 4 bytes\n",
+    "data_file.seek(50) # go to position 50\n",
+    "print(data_file.read(10)) # print 10 more bytes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And we can read the data directly into a byte array."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b = bytearray(100) # create a byte array of length 100\n",
+    "data_file.readinto(b) # read 100 bytes into the byte array\n",
+    "b[:]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calling `getbuffer` returns the location in memory where the dataset is being stored locally."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b = data_file.getbuffer()\n",
+    "b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can use the memory buffer to make local writes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_file.seek(100) # go to position 100\n",
+    "b[100:110] = b\"helloworld\"; # write 'helloworld' into our local data\n",
+    "data_file.seek(100) # return to position 100\n",
+    "n = data_file.read(10) # read back the written bytes\n",
+    "n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have opened a remote dataset and read parts of it using random access! Use `remote_open` when you want access to the raw data in a dataset, e.g., if you have Python code to read bytes in a particular format.\n",
+    "\n",
+    "*Note:* Without some prior knowledge about the format of the dataset, `remote_open` is not an effective method of parsing data. Since we are reading a raw file object, we need to know layout of the data and the data types (e.g. ints, floats, etc.). To read a dataset as a netCDF object, use [`remote_access`](https://unidata.github.io/siphon/latest/api/catalog.html?highlight=remote%20open#siphon.catalog.Dataset.remote_access)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## See also\n",
```

Comment:
Considering you do mention the Siphon Remote Access notebook at the end of the previous section, I think it would be a good idea to include it here as well in a ###Related Notebooks subsection.